### PR TITLE
Upgrade node-oidc-client to version ^1.01

### PR DESCRIPTION
**Pull request by francisco-idpartner**
Upgrade node-oidc-client to version ^1.01

### CODE REVIEW

**Review by yovasx2 — APPROVED** [review]

File: routes/index.js [hunk 1]
```diff
@@ -31,16 +31,15 @@ router.get('/jwks', async (req, res, next) => {
 router.get('/auth', async (req, res, next) => {
   const scope = ['openid', 'email', 'profile'];
   req.session.idp_proofs = idPartnerClient.generateProofs();
+  req.session.issuer = req.query.iss;
+  req.session.saved_provider_id = req.query.saved_provider_id;
```
Comment (chrismccaw) [inline routes/index.js:35]: Why is this needed?
